src/identity-providers/ManageOrderDialog.tsx

```diff
@@ -111,28 +112,23 @@ export const ManageOrderDialog = ({
         onDragCancel={onDragCancel}
         itemOrder={order}
       >
-        {sortBy(providers, "config.guiOrder").map((provider) => (
+        {sortBy(providers, "config.guiOrder", "alias").map((provider) => (
           <DataListItem
-            aria-labelledby={provider.alias}
-            id={`${provider.alias}-item`}
+            aria-label={provider.alias}
+            id={provider.alias}
             key={provider.alias}
           >
             <DataListItemRow>
               <DataListControl>
-                <DataListDragButton
-                  aria-label="Reorder"
-                  aria-labelledby={provider.alias}
-                  aria-describedby={t("manageOrderItemAria")}
-                  aria-pressed="false"
-                />
+                <DataListDragButton aria-label={t("manageOrderItemAria")} />
               </DataListControl>
               <DataListItemCells
                 dataListCells={[
                   <DataListCell
-                    key={`${provider.alias}-cell`}
+                    key={provider.alias}
                     data-testid={provider.alias}
                   >
-                    <span id={provider.alias}>{provider.alias}</span>
+                    <span>{provider.alias}</span>
```

Comment:
Do we still need this `span` element?
```suggestion
                    {provider.alias}
```